### PR TITLE
# Add Animation tokens

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/245345663/pulses/2647297506?userId=28252820
- Add Registration Timing tokens.
- Add Registration Duration tokens.
- Add Base Timing Tokens.
- Add Base Duration Tokens.
- Assign proper Base Timing Token to Common "expand-animation-timing" Key.

### CODE REVIEW

File: stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ const FONT_LINE_HEIGHTS = [
   "--font-line-height-subtext",
 ];
 
+const ANIMATION_TIMING = ["--animation-timing-enter", "--animation-timing-exit", "--animation-timing-transition", "--animation-timing-emphasize", "--expand-animation-timing"];
+
+const ANIMATION_DURATION = ["--animation-productive-short", "--animation-productive-medium", "--animation-productive-medium", "--animation-productive-long", "--animation-expressive-short", "--animation-expressive-medium", "--animation-expressive-long"];
```
Comment (laviomri) [inline stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js:53]: nitpicking: other arrays of CSS var names are in plural
```suggestion
const ANIMATION_TIMINGS = ["--animation-timing-enter", "--animation-timing-exit", "--animation-timing-transition", "--animation-timing-emphasize", "--expand-animation-timing"];

const ANIMATION_DURATIONS = ["--animation-productive-short", "--animation-productive-medium", "--animation-productive-medium", "--animation-productive-long", "--animation-expressive-short", "--animation-expressive-medium", "--animation-expressive-long"];
```

**Review by laviomri — APPROVED** [review]
looks good to me! :)

File: stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js [hunk 2]
```diff
@@ -109,7 +115,8 @@ const PROPS_TO_ALLOWED_VARS = {
   ...mapPropsToAllowedVars("line-height", FONT_LINE_HEIGHTS),
   ...mapPropsToAllowedVars("font-weight", FONT_WEIGHTS),
   ...mapPropsToAllowedVars("font-size", FONT_SIZES),
-  ...mapPropsToAllowedVars(TIMING_FUNCTION_PROPS, "--expand-animation-timing"),
+  ...mapPropsToAllowedVars(TIMING_FUNCTION_PROPS, ANIMATION_TIMING),
```
Comment (orrgottlieb) [inline stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js:118]: `ANIMATION_TIMING` this is not defined - should be `ANIMATION_TIMINGS`

**Review by orrgottlieb — CHANGES_REQUESTED** [review]

File: stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js (hunk 2 above)
Comment (3dyonic) [reply stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js:118]: @laviomri advised to correct the naming conventions, the const reference got away :)
Thanks

Comment (3dyonic) [reply stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js:120]: we changed to "ANIMATION_TIMING" for naming conventions consistency

**Comment by 3dyonic** [comment]
made a lot of mess in commits. reverted them. now all is good.

**Review by orrgottlieb — APPROVED** [review]

**Pull request merged**